docs/steps-rest.md

```diff
@@ -0,0 +1,81 @@
+---
+id: steps-rest
+title: Rest api
+---
+
+# Steps used for testing REST api:
+
+In order to configure url for api please change `apiUrl` field in `functional-tests/kakunin.conf.js`. This will set baseUrl of application api.
+
+## `I send ":methodName" request on ":endpoint" endpoint`
+
+Sends given request method to given website endpoint.
```

Comment:
I'm not an English expert but I think it should be `to the given website endpoint`. In this step and the following ones.